orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineExpressionFunctionProvider.java

```diff
@@ -108,4 +117,23 @@ public static String pipelineId(Execution execution, String pipelineName) {
       throw new SpelHelperFunctionException("Failed to evaluate #pipelineId function", e);
     }
   }
+
+  /**
+   * Function to convert pipeline name to pipeline ID (within current application), will return Null
+   * if pipeline ID not found
+   *
+   * @param execution the current execution
+   * @param pipelineName name of the pipeline to lookup
+   * @return the id of the pipeline or null if pipeline not found
+   */
+  public static String pipelineIdOrNull(PipelineExecution execution, String pipelineName) {
```

Comment:
@marchello2000 seems the type `Execution` was changed to `PipelineExeuction` in 1.20. I can't cherry-pick this :(